layers/+lang/scheme/packages.el

```diff
@@ -102,17 +102,44 @@
         "sR" 'geiser-eval-region-and-go
         "ss" 'geiser-set-scheme)
 
+      (evil-define-key 'insert 'geiser-repl-mode-map
+        (kbd "S-<return>") 'geiser-repl--newline-and-indent)
+
       (evil-define-key 'normal 'geiser-repl-mode-map
-      "gj" 'geiser-repl-next-prompt
-      "gk" 'geiser-repl-previous-prompt)
+        "]]" 'geiser-repl-next-prompt
+        "[[" 'geiser-repl-previous-prompt
+        "gj" 'geiser-repl-next-prompt
+        "gk" 'geiser-repl-previous-prompt)
 
+      (spacemacs/declare-prefix-for-mode 'geiser-repl-mode "mh" "help")
       (spacemacs/set-leader-keys-for-major-mode 'geiser-repl-mode
-        "q" 'geiser-repl-exit
+        "b" 'geiser-pop-symbol-stack
         "c" 'geiser-repl-clear-buffer
         "i" 'geiser-repl-interrupt
+        "f" 'geiser-load-file
+        "l" 'geiser-insert-lambda
         "m" 'geiser-repl-import-module
         "u" 'geiser-repl-unload-function
-        "h" 'geiser-doc-symbol-at-point))))
+        "hd" 'geiser-doc-symbol-at-point
+        "s" 'geiser-squarify
```

Comment:
Is this a common part of workflow?

Some other modes may also accomplish this functionality, eg in parinfer-rust-mode if you change the opening parenthesis to bracket it changes the closing one (or vice versa).

p.s. one real issue is the command name itself is really not informative...

Reply:
Well it is `almost` provided by `sp-rewrap-sexp` or `evil-surround` (`c s ) ]`). But they both require you to type explicitly the new type of enclosing.
I know in racket and chez, brackets are equivalent to parenthesis (they are used interchangeably to "improve" readability. I don't know about scheme in general). So anyway, I guess this shortcut is quite handy.